ssg/entities/profile_base.py

```diff
@@ -118,6 +119,8 @@ def to_xml_element(self):
 
         element = ET.Element('{%s}Profile' % XCCDF12_NS)
         element.set("id", OSCAP_PROFILE + self.id_)
+        if self.status:
+            add_sub_element(element, "status", XCCDF12_NS, str(self.status))
```

Comment:
I suggest testing this feature by a unit test. 

You can extend or reuse this test:
https://github.com/ComplianceAsCode/content/blob/80b195e4849bacadad558e2ce531e211dcb91eb4/tests/unit/ssg-module/test_build_yaml.py#L507

Reply:
Tests addded, thank you for the review.